include/common/enum.inc

```diff
@@ -881,6 +881,9 @@ E(SharedMemoryNoMax, 0x011A, "shared memory must have maximum")
 E(IntrinsicsTableNotFound, 0x011B, "intrinsics table not found")
 // Malformed table (Ref-types proposal)
 E(MalformedTable, 0x011C, "malformed table")
+// The store alignment must be less than 128 and multi-memory 
```

Comment:
I think the comment can be compressed into 1 line:

`// Alignment must < 64 without and < 128 with multi-memory proposal.`